.github/workflows/build.yaml

```diff
@@ -0,0 +1,24 @@
+name: Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['2.7', '3.6']
```

Comment:
Are you just wanting to get the build working as is and leaving this for a subsequent update to 3.8

Reply:
I am preparing another PR to drop python 2 and adding more python 3 versions to the build